### DISCUSSION TITLE
How to configure MSS?

### DISCUSSION THREAD

**Discussion by gou4shi1** [Haivision/srt#2096] [Q&A] [answered]
For example, in 4G network, live mode and message mode.

**Comment by maxsharabayko** [comment, marked as answer]
MSS can be configured using the [`SRTO_MSS`](https://github.com/Haivision/srt/blob/master/docs/API/API-socket-options.md#SRTO_MSS) socket option.
The value is then sent in the handshake (see the [Maximum Transmission Unit Size](https://datatracker.ietf.org/doc/html/draft-sharabayko-srt-01#section-3.2.1) field). The smaller value of the two peers is used once a connection is established.
